core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -549,13 +549,15 @@ public SsccCommitResponse call(SsccRegionService instance) throws IOException {
                retry = false;
              }
           }
-          catch (UnknownTransactionException ute) {
-              LOG.error("Got unknown exception in doCommitX by participant " + participantNum
-            		  + " for transaction: " + transactionId + " " + ute);
-              transactionState.requestPendingCountDec(true);
-              throw new UnknownTransactionException();
-          }
           catch (Exception e) {
+             if(e.toString().contains("UnknownTransactionException")) {
```

Comment:
Is it conceivable that we might get a false positive with a string lookup? (Could the characters "UnknownTransactionException" occur for some other reason in the text, for example as part of a delimited identifier for a Trafodion table name?)